### DISCUSSION TITLE
Noob question #2: best pratice in a production environment

### DISCUSSION THREAD

**Discussion by LuizCesarLeite** [strapi/strapi#6950] [Legacy Topics]
Hi there again!

In my local machine I'm using "npm run develop".

In my server I'll use pm2, but what is the best pratice for it? Use "npm run develop" and after "npm run build"?

Thanks a lot!

**Comment by derrickmehaffy** [comment]
Hello, please see the related guide to using PM2: https://strapi.io/documentation/v3.x/guides/process-manager.html#process-manager
